recipes/tket/conanfile.py

```diff
@@ -64,6 +65,7 @@ class TketConan(ConanFile):
         "Characterisation",
         "Converters",
         "TokenSwapping",
+        "TokenSwappingWithArch",
```

Comment:
Can we put these source files inside the `Architecture` component, as it is just a thin connecting layer. I think this makes more sense than a whole new component.

Reply:
Yes that's fine.